components/textbook-beta/SummerSchoolCoursePages.vue

```diff
@@ -0,0 +1,340 @@
+<template>
```

Comment:
Have you checked if you can reuse the `CoursePagesSection` component instead of creating this `SummerSchoolCoursePagesSection`?

Reply:
Same here, 

I started to turn this into a reusable component but with this task being a `S`, adding refactors into this felt out of scope... so I created an issue to handle extending the component https://github.com/Qiskit/qiskit.org/issues/2168